rmf/rmf-java-base/src/main/java/io/vrap/rmf/base/client/http/RequestPolicyBuilder.java

```diff
@@ -0,0 +1,95 @@
+
+package io.vrap.rmf.base.client.http;
+
+import java.time.Duration;
+import java.util.*;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+import io.vrap.rmf.base.client.ApiHttpRequest;
+import io.vrap.rmf.base.client.ApiHttpResponse;
+
+import dev.failsafe.Policy;
+import dev.failsafe.spi.Scheduler;
+
+/**
+ * <h2>PolicyBuilder</h2>
```

Comment:
```suggestion
 * <h2>RequestPolicyBuilder</h2>
```

I am guessing this text needs an update. 😄